### ISSUE TITLE
Extend welcome message in DTS to provide more information

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#540]
### The problem you're addressing (if any)

Some issue reports regarding DTS malfunction contain pictures of the screen. This is probably because it is easiest way to provide feedback for us.

### Describe the solution you'd like

Please add critical information to the Dasharo welcome message, ideally below the DTS version:
- BIOS version
- mainboard version
- CPU model
- amount and model of memory?

We cannot pollute the screen too much because it would be unreadable.

### Where is the value to a user, and who might that user be?

 If we had some important information from `dmidecode` on this screen, we could avoid asking additional questions and shorten the feedback loop.

### Describe alternatives you've considered

The alternative would be the introduction of the MrChromebox look&feel interface, which consists of some information by default, e.g., it can detect some protection bits.

**Comment by mkopec** [comment]
> BIOS version
> mainboard version
> CPU model
> amount and model of memory?

these could be printed as a QR code for easy copying from DTS :)

**Comment by pietrushnic** [comment]
:scream: I don't know why, but I have very little trust in QR codes. It is a very nice way to deliver zero days.

**Comment by DaniilKl** [comment]
@pietrushnic From my poin of view following information may be useful in welcome message:

  * BIOS
    * Firmware manufactorer, can be gotten from `dmidecode/BIOS Information/Vendor`:
    * Firmware version, can be gotten from `dmidecode/BIOS Information/Version`:
    * BIOS revision, `dmidecode/BIOS Information/BIOS revision`:
  * CPU
    * Manufacturer, can be gotten from `dmidecode/Processor Information/Version`:
    * Revision, can be gotten from `dmidecode/Processor Information/Version`:
    * Current frequency, can be gotten from `dmidecode/Processor Information/Version`:
    * Current voltage, can be gotten from `dmidecode/Processor Information/Voltage`:
  * Memory
    * Amount of memory being used, sum of size for each used bank:
    * For each used bank `dmidecode/Memory Device/Bank`:
      * Manufactorer `dmidecode/Memory Device/Manufacturer`:
      * Part number, `dmidecode/Memory Device/Part Number`:
      * Type, `dmidecode/Memory Device/Type`:
      * Size, `dmidecode/Memory Device/Size`:
  * Motherboard
    * Manufacturer, can be gotten from `dmidecode/Base Board Information/Manufacturer`: 
    * Name, can be gotten from `dmidecode/Base Board Information/Product Name`: 
    * Version, can be gotten from `dmidecode/Base Board Information/ Version`: 

An it can be printed in one line for each part format, for example:

```
BIOS: ...
CPU: ...
Motherboard: ...
RAM: ..., #1 ..., #2 ...
```

What do you think?

**Comment by macpijan** [comment]
I already have done it to some extent: https://github.com/Dasharo/dasharo-issues/discussions/361#discussioncomment-5177841

Code is link there to be reused.

**Comment by pietrushnic** [comment]
@DaniilKl I agree, but I would really like to see better UI in form of MrChromebox style design as @macpijan did, before committing to much time into this issue.

**Comment by pietrushnic** [comment]
> Although I like the way @macpijan implemented this, I like the current minimalistic DTS welcome message more than the MrChromebox style. So, I presented hardware inf. in the same minimalistic way.

This is typical for our judgment and can be classified as https://en.wikipedia.org/wiki/False_consensus_effect
It cannot be our driving factor in design decisions.

> @pietrushnic, to clarify the situation, should I wait for the style of the DTS welcome message to be changed, change the style by myself, or add hardware inf. in the way I proposed above?

Let's discuss that internally.

**Comment by macpijan** [comment]
This must come after (together) with the: https://github.com/Dasharo/dasharo-issues/issues/225

We should not extend the current UI much more.
